packages/amplify-graphql-api-construct-tests/src/__tests__/backends/sql-models/app.ts

```diff
@@ -1,32 +1,26 @@
 #!/usr/bin/env node
 import 'source-map-support/register';
-import { App, Stack, Duration, CfnOutput, RemovalPolicy } from 'aws-cdk-lib';
+import { App, Stack, Duration, CfnOutput } from 'aws-cdk-lib';
```

Comment:
This file is result of splitting the `sql-models.test.ts`.

Changes in this file are to reset it's state to match `feature/gen2-release` branch.